src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -8,21 +8,6 @@ import {
   Prospect,
   UrlMetadata,
 } from '../../api/generatedTypes';
-/**
- *
- * This is a helper file that contains some helper functions. Right now
- * it only has one but we could add more to it
- */
-/**
- *
- * This is a helper file that contains some helper functions. Right now
- * it only has one but we could add more to it
- */
-/**
- *
- * This is a helper file that contains some helper functions. Right now
- * it only has one but we could add more to it
- */
 
```

Comment:
Unrelated to the main ticket - deleting this set of comments in the helper functions file which has half a dozen helper functions by now.